src/plugins/subsystems/drm/drm.sh

```diff
@@ -75,6 +76,10 @@ function drm_main()
     # For unload DRM drivers, we need to make sure that we turn off user GUI
     [[ "$gui_off" != 1 ]] && gui_control 'OFF' "$target" "$remote"
     module_control 'UNLOAD' "$target" "$remote" "$unload_module" "$flag"
+    if [[ "$?" != 0 ]]; then
+      return 22 # EINVAL
```

Comment:
You should add something here if `module_control` does not output any failure message.

Reply:
I checked the `module_control` function and it does print an error message when it returns 22, but it can also call the `cmd_remotely` function, which in turn calls `cmd_manager` and so on. Following David's suggestion, to make things easier I've added a more detailed error message in drm.sh to avoid the error message not being printed